website/app/components/doc/icons-list/meta-row.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="doc-icons-list-grid-item__meta-row">
+  <div class="doc-icons-list-grid-item__meta-label">{{@label}}</div>
+  <div class="doc-icons-list-grid-item__meta-value">
+    <Doc::CopyButton @textToCopy={{@value}} @type="ghost" />
```

Comment:
May be unavoidable, but allowing people to copy the keywords doesn't seem that useful
<img width="271" alt="Screen Shot 2022-12-09 at 11 43 00" src="https://user-images.githubusercontent.com/1672302/206761233-653e7752-9016-4dbd-b63f-9c3f11492257.png">

Given how much space things are taking up in the design I wonder if they are actually worth showing at all? They're more of an internal thing than something consumers would care about.

Reply:
Pushed